packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.test.tsx

```diff
@@ -246,4 +255,59 @@ describe('VerticalStackedBarChart - mouse events', () => {
     const tree = toJson(wrapper, { mode: 'deep' });
     expect(tree).toMatchSnapshot();
   });
+
+  it('Should render callout correctly on mousemove', async () => {
+    wrapper = mount(
+      <VerticalStackedBarChart data={chartPoints} calloutProps={{ doNotLayer: true }} enabledLegendsWrapLines />,
+    );
+    await new Promise(resolve => setTimeout(resolve));
+    wrapper.update();
+    wrapper.find('rect').at(2).simulate('mousemove');
```

Comment:
what does 2 and 3 represent

Reply:
There are multiple rect in the chart, so .at(2) and .at(3) selects the 3rd and 4th rect respectively